.Rbuildignore

```diff
@@ -20,3 +20,4 @@
 # CRAN stuff
 ^cran-comments\.md$
 ^NEWS\.md$
+^vignettes/dummynetcdf\.nc$
```

Comment:
Need to ignore `dummynetcdf.nc` wherever it was created and not cleaned up including rSW2analysis/ or rSW2analysis/R/ depending from which location `devtools::run_examples()` was run -- or at least until the example code cleans up

Reply:
added to ignore file